_posts/2019-05-24-varying-slopes/varying_slopes.Rmd

```diff
@@ -0,0 +1,748 @@
+---
+title: "Hierarchical partial pooling, continued: Varying slopes models with TensorFlow Probability"
+description: > 
+   This post builds on our recent introduction to multi-level modeling with tfprobability, the R wrapper to TensorFlow Probability. We show how to pool not just mean values ("intercepts"), but also relationships ("slopes"), thus enabling models to learn from data in an even broader way. Again, we use an example from Richard McElreath's "Statistical Rethinking"; and again, terminology as well as the way we present this topic are largely owed to this book.
```

Comment:
I think you can cut the second `and again,` in this paragraph and replace it with "the", perhaps?